include/oneapi/dpl/complex

```diff
@@ -24,6 +24,149 @@ namespace oneapi
 namespace dpl
 {
 using ::std::complex;
+
+using ::std::arg;
+using ::std::conj;
+using ::std::exp;
```

Comment:
::std::exp has C - overloads as well.. why do we have "using", but  for the other functions we have C++ specializations for "complex"  below?

Reply:
fixed.